src/v2/components/IconButton.jsx

```diff
@@ -29,10 +29,17 @@ import concatClassNames from '../modules/concatClassNames.mjs'
  * @returns {React.Element}
  */
 export default function IconButton(props) {
-  const {className, disabled, icon, label, ...otherProps} = props
+  const {className, disabled, href, icon, label, ...otherProps} = props
   const classNames =
     concatClassNames(className, 'c7e-icon-button', disabled && 'c7e-icon-button--disabled')
 
+  if (!!href) {
```

Comment:
I guess it's more a style thing, but another option would be:
if (href) {
  return xxx;
} else {
  return yyy;
}